2025-03-13 10:31:50,276 INFO    MainThread:3873852 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-03-13 10:31:50,276 INFO    MainThread:3873852 [wandb_setup.py:_flush():67] Configure stats pid to 3873852
2025-03-13 10:31:50,276 INFO    MainThread:3873852 [wandb_setup.py:_flush():67] Loading settings from /home/javitrucas/.config/wandb/settings
2025-03-13 10:31:50,277 INFO    MainThread:3873852 [wandb_setup.py:_flush():67] Loading settings from /home/javitrucas/TFG/notebooks/wandb/settings
2025-03-13 10:31:50,277 INFO    MainThread:3873852 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-13 10:31:50,277 INFO    MainThread:3873852 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/javitrucas/TFG/notebooks/wandb/run-20250313_103150-4xgm3qbf/logs/debug.log
2025-03-13 10:31:50,277 INFO    MainThread:3873852 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/javitrucas/TFG/notebooks/wandb/run-20250313_103150-4xgm3qbf/logs/debug-internal.log
2025-03-13 10:31:50,277 INFO    MainThread:3873852 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f4dc55642d0>
2025-03-13 10:31:50,278 INFO    MainThread:3873852 [wandb_init.py:init():761] calling init triggers
2025-03-13 10:31:50,278 INFO    MainThread:3873852 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'target_digit': 3, 'bag_size': 10, 'num_epochs': 1, 'learning_rate': 0.001, 'pooling_type': 'attention', '_wandb': {}}
2025-03-13 10:31:50,278 INFO    MainThread:3873852 [wandb_init.py:init():784] starting backend
2025-03-13 10:31:50,278 INFO    MainThread:3873852 [wandb_init.py:init():788] sending inform_init request
2025-03-13 10:31:50,282 INFO    MainThread:3873852 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-13 10:31:50,283 INFO    MainThread:3873852 [wandb_init.py:init():803] backend started and connected
2025-03-13 10:31:50,286 INFO    MainThread:3873852 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-13 10:31:50,288 INFO    MainThread:3873852 [wandb_init.py:init():896] updated telemetry
2025-03-13 10:31:50,299 INFO    MainThread:3873852 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-03-13 10:31:50,710 INFO    MainThread:3873852 [wandb_init.py:init():995] starting run threads in backend
2025-03-13 10:31:50,902 INFO    MainThread:3873852 [wandb_run.py:_console_start():2377] atexit reg
2025-03-13 10:31:50,903 INFO    MainThread:3873852 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-13 10:31:50,903 INFO    MainThread:3873852 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-13 10:31:50,903 INFO    MainThread:3873852 [wandb_run.py:_redirect():2317] Redirects installed.
2025-03-13 10:31:50,907 INFO    MainThread:3873852 [wandb_init.py:init():1037] run started, returning control to user process
2025-03-13 10:32:37,802 INFO    MainThread:3873852 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-13 10:32:37,802 INFO    MainThread:3873852 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-13 10:36:30,293 INFO    MainThread:3873852 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-13 10:36:30,303 INFO    MainThread:3873852 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-13 10:36:30,304 INFO    MainThread:3873852 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-13 10:37:08,707 INFO    MainThread:3873852 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-13 10:37:08,716 INFO    MainThread:3873852 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-13 10:37:08,717 INFO    MainThread:3873852 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-13 10:37:12,217 INFO    MainThread:3873852 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-13 10:37:38,378 WARNING MsgRouterThr:3873852 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "/home/javitrucas/miniconda3/envs/tfg/lib/python3.11/site-packages/wandb/sdk/interface/router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/javitrucas/miniconda3/envs/tfg/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/javitrucas/miniconda3/envs/tfg/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/javitrucas/miniconda3/envs/tfg/lib/python3.11/site-packages/wandb/sdk/interface/router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/javitrucas/miniconda3/envs/tfg/lib/python3.11/site-packages/wandb/sdk/interface/router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-03-13 10:37:38,396 INFO    MsgRouterThr:3873852 [mailbox.py:close():115] Closing mailbox, abandoning 1 handles.
2025-03-13 10:37:38,702 INFO    MainThread:3873852 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-13 10:37:38,702 INFO    MainThread:3873852 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-13 10:37:39,538 INFO    MainThread:3873852 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-13 10:37:40,149 INFO    MainThread:3873852 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-13 10:37:40,149 INFO    MainThread:3873852 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-13 10:37:48,110 INFO    MainThread:3873852 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-13 10:37:48,273 INFO    MainThread:3873852 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-13 10:37:48,273 INFO    MainThread:3873852 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-13 10:38:18,417 INFO    MainThread:3873852 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-13 10:38:18,633 INFO    MainThread:3873852 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-13 10:38:18,633 INFO    MainThread:3873852 [wandb_init.py:_pause_backend():564] pausing backend
